tests/representation/test_feature.py

```
@@ -0,0 +1,113 @@
+import numpy as np
+import pandas as pd
+import pytest
+from iohub import open_ome_zarr
+
+from viscy.representation.evaluation.feature import (
+    CellFeatures,
+    DynamicFeatures,
+)
+
+
+@pytest.mark.parametrize("channel_idx", [0, 1])
+def test_cell_features_with_labels_hcs(
+    small_hcs_dataset, small_hcs_labels, channel_idx
+):
+    """Test CellFeatures with labels HCS dataset."""
+    data_path = small_hcs_dataset
+    with open_ome_zarr(data_path) as dataset:
+        _, position = next(dataset.positions())
+        image_array = position["0"]
+
+    with open_ome_zarr(small_hcs_labels) as labels_dataset:
+        _, position = next(labels_dataset.positions())
+        labels_array = position["0"]
+
+    # Extract patch from center
+    patch_size = 16
+    t, z = 0, 0
+    y_center, x_center = image_array.shape[-2] // 2, image_array.shape[-1] // 2
+    half_patch = patch_size // 2
+
+    y_slice = slice(y_center - half_patch, y_center + half_patch)
+    x_slice = slice(x_center - half_patch, x_center + half_patch)
+
+    image_patch = image_array[t, channel_idx, z, y_slice, x_slice]
+    labels_patch = labels_array[t, channel_idx, z, y_slice, x_slice]
+
+    cf = CellFeatures(
+        image=image_patch.astype(np.float32), segmentation_mask=labels_patch
+    )
+    features_df = cf.compute_all_features()
+
+    assert isinstance(features_df, pd.DataFrame)
+    assert len(features_df) == 1
+
+    assert "mean_intensity" in features_df.columns
+    assert "contrast" in features_df.columns
+    assert "zernike_std" in features_df.columns
+    assert "area" in features_df.columns
+
+    for col in features_df.columns:
+        value = features_df[col].iloc[0]
+        if col in ["kurtosis", "skewness"]:
+            if not np.isfinite(value):
+                print(f"Warning: {col} is {value} for channel {channel_idx}")
```

What are these print statements for?

---

If the things it's warning about shouldn't happen, it should raise an error.